showcase/src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,30 @@
+import { ExampleViewerComponent } from "./example-viewer/example-viewer.component";
+import * as hljs from "highlight.js";
+import { HighlightJsModule, HIGHLIGHT_JS } from "angular-highlight-js";
+import { MatButtonModule, MatIconModule, MatTabsModule, MatTooltipModule, MatSnackBarModule } from "@angular/material";
+import { CommonModule } from "@angular/common";
+import { NgModule } from "@angular/core";
+
+export function highlightJsFactory(): any {
+	return hljs;
+}
+
+@NgModule({
+	imports: [
+		HighlightJsModule.forRoot({
+			provide: HIGHLIGHT_JS,
+			useFactory: highlightJsFactory
+		}),
+		MatButtonModule,
+		MatIconModule,
+		MatTooltipModule,
+		MatSnackBarModule,
+		MatTabsModule,
+		CommonModule
+	],
+	providers: [],
```

Comment:
Question, isn't the service to fetch the files supposed to be included in here? I think it has not been defined in any module right? or am I missing something?

Reply:
Doesn t make any difference for now but true it is more correct to provide it